### PR TITLE
Web validator

### PR DESCRIPTION
**Pull request by aliparslan**
**Info**
This PR has the foundations for the web validator built on top of Eido. It comes with working `GET` and `POST` endpoints and the schema-specific validation page. The other features, including the universal validator and a couple other endpoints, are in also place, but do not work consistently (or at all). This is the first of two pull requests, the latter of which will have the web validator in a stable and finished state.

### CODE REVIEW

**Comment by nsheff** [comment]
Hey @aliparslan -- I think this is fine and we need to just merge this in, but I have a few things I'd like you to change first:

1. Don't put spaces in HTML file names. These are going to be used in URLs, etc -- just use underscores. Go through all your files and make sure there's no spaces in filenames.
2. There are no docstrings on your API endpoint functions. Can you please add docstrings to every new function you've written, explaining 1. what the function is supposed to do; 2. define the inputs; 3. define the output.
3. Please clean up old stuff -- there's no need to store things like `.orig` files, these can always be retrieved from prior github versions.
4. I can't follow why some of the HTML templatse have names like 'schema_LOCAL_36579.html' -- what is meant by the number 36579, and what is LOCAL, or BASE, or REMOTE? can you clarify these filesnames, clean this up so that it's more understandable for another developer?

**Review by nsheff — CHANGES_REQUESTED** [review]
See above for requested changes

**Comment by nleroy917** [comment]
I'll review + pull these in once we get the `pepdbagent` issue sorted out: https://github.com/pepkit/pepdbagent/issues/48#issuecomment-1309003043

File: pephub/routers/eido.py [hunk 1]
```diff
@@ -139,13 +280,18 @@ async def validate_pep(
...
-#     print(je.list_templates())
-#     return HTMLResponse(je.get_template("index.html").render())
+#     return HTMLResponse(je.get_template("schema.html").render())
 
 
 @router.get("/validator")
 async def main():
     print(je.list_templates())
-    return FileResponse(os.path.join(STATICS_PATH, "index.html"))
+    return FileResponse(os.path.join(STATICS_PATH, "index2.html"))
```
Comment (nleroy917) [inline pephub/routers/eido.py:297]: Can we delete `index.html` and rename `index2.html` to `index.html`? Does that make sense?

File: requirements/requirements-all.txt [hunk 2]
```diff
@@ -1,6 +1,6 @@
 peppy>=0.32
 fastapi>=0.72.0
-eido>=0.1.5
+eido @ git+https://github.com/pepkit/eido@dev_alip#egg=eido
```
Comment (nleroy917) [inline requirements/requirements-all.txt:3]: Are we still pointing at the old `eido`?

**Review by nleroy917 — CHANGES_REQUESTED** [review]

**Pull request merged**